src/main/java/frc/robot/SensorInputs.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot;
+
+import com.kauailabs.navx.frc.AHRS;
+
+import edu.wpi.first.wpilibj.SPI.Port;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class SensorInputs {
+    //Sensor Definitions
+    private final AHRS navxAhrs = new AHRS(Port.kMXP);
```

Comment:
Here you're using bindings from a C++ library, and Hungarian notation _is_ much more common in C++ code. That said, its usage has been declining since its peak in the 1990s, maybe early 2000s.

Reply:
(Note: There's nothing to be done here, this is just FYI)